<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_NewBattleMap</class>
 <widget class="QDialog" name="Dialog_NewBattleMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>New Battle Map</string>
  </property>
  <widget class="QDialogButtonBox" name="DialogButtonBox">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>850</y>
     <width>1150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="GraphicsView_BattleMap">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>200</y>
     <width>1150</width>
     <height>650</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="interactive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="RadioButton_SourceBattleMap">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>25</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Battle Map image from source</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="RadioButton_EmptyBattleMap">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>55</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Battle Map image empty</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckBox_DrawBattleMapGrid">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>95</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw Battle Map grid</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_NumberRows">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>135</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of rows</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_NumberColumns">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>165</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of columns</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_NumberRows">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>135</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_NumberColumns">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>165</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButton_DecrementNumberRows">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>135</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButton_IncrementNumberRows">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>135</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButton_DecrementNumberColumns">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>165</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButton_IncrementNumberColumns">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>165</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_Source">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>25</y>
     <width>750</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButton_SelectSource">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1075</x>
     <y>25</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Select source</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
